src/control/lib/control/auto.go

```diff
@@ -554,6 +559,41 @@ func defaultEngineCfg(idx int) *engine.Config {
 		WithBdevDeviceList([]string{}...)
 }
 
+func getHugePageSize() (int, error) {
```

Comment:
please use the helper in src/control/server/hugepages.go and move it out into the common package, just to reduce duplication, thanks